.github/workflows/build.yaml

```diff
@@ -2,9 +2,9 @@ name: build
 on: [push, pull_request]
 jobs:
   build:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     container:
-      image: docker://ros:eloquent-ros-base-bionic
+      image: docker://ros:foxy-ros-base-focal
       
     steps:
     - name: deps
```

Comment:
I guess this step can be removed as well?

Reply:
Are you suggesting getting rid of the rosdep install? I'm not sure if `libfcl` and `libccd` are included in the base image of foxy :thinking: 

But even if they are, I think the pipeline is more robust to future dependency changes if we just keep this in.

Reply:
I meant the normal "deps" step that only installs g++8, but it probably doesn't matter that much

Reply:
Oh sorry I got confused because GitHub had collapsed that part of the file, so it looked like you were referring to `rosdep install` :facepalm: